data/CMakeLists.txt

```diff
@@ -186,6 +186,12 @@ if(USE_XMLLINT)
   add_dependencies(darktablerc_file validate_darktableconfig_xml)
 endif(USE_XMLLINT)
 
+if (WIN32 AND EXISTS ${CMAKE_CURRENT_BINARY_DIR}/darktablerc)
+  #make sure line endings are UNIX like even on Windows
+  configure_file(${CMAKE_CURRENT_BINARY_DIR}/darktablerc ${CMAKE_CURRENT_BINARY_DIR}/darktablerc.tmp NEWLINE_STYLE LF)
```

Comment:
My understanding is that darktablerc is a result of an **xsltproc** custom command. 
```
  add_custom_command(
    DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/../tools/generate_darktablerc.xsl ${CMAKE_CURRENT_BINARY_DIR}/darktableconfig.dtd ${CMAKE_CURRENT_BINARY_DIR}/darktableconfig.xml
    OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/darktablerc
    COMMAND ${Xsltproc_BIN} ${CMAKE_CURRENT_SOURCE_DIR}/../tools/generate_darktablerc.xsl ${CMAKE_CURRENT_BINARY_DIR}/darktableconfig.xml > ${CMAKE_CURRENT_BINARY_DIR}/darktablerc
  )
```
I tried to change the xsl file to make sure darktablerc have LF only at the line ends but did not find a good solution, and ended always with CRLF line ends. Any suggestion would be helpful here.